### PR TITLE
Tests for CRUD operations on application services

### PR DESCRIPTION
**Pull request by mvshao**
**Description**

As part of the `fast integration test`, applications services need to be created, updated, deleted.

Changes proposed in this pull request:

- adding tests for CRUD operations on applications services

**Related issue(s)**
#11770

### CODE REVIEW

File: tests/fast-integration/test/1-commerce-mock.js [hunk 1]
```diff
@@ -57,6 +60,12 @@ function commerceMockTests() {
       await checkFunctionResponse(testNamespace);
     });
 
+    it("should add, update and delete a service", async function () {
```
Comment (franpog859) [inline tests/fast-integration/test/1-commerce-mock.js:63]: Cool! The only thing is I'd move it after the lambda creation not to break the order of the main flow

File: tests/fast-integration/package-lock.json [hunk 2]
```diff
@@ -1,4446 +1,8 @@
 {
   "name": "kyma-js",
   "version": "0.0.1-alpha.12",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (franpog859) [inline tests/fast-integration/package-lock.json:4]: Try to upgrade the version of your `node.js`. I think we don't want to downgrade the version here

**Review by franpog859 — DISMISSED** [review]
LGTM. Please, ask owners about the `package-lock.json` changes. These deletions seem quite suspicious

**Review by rakesh-garimella — APPROVED** [review]

**Pull request merged**